src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/forumpageobject/ForumBoardPage.java

```diff
@@ -168,9 +167,11 @@ public void startDiscussionWithVideo(String url, String title) {
   }
 
   public void unfollowIfDiscussionIsFollowed(int threadNumber) {
-    WebElement followButton =
-        driver.findElement(By.cssSelector(".thread:nth-child(" + threadNumber + ") li.follow"));
-    wait.forElementVisible(followButton);
+
+    //TODO: Get a list of threads not to put locator in this method
+    By followButtonBy = By.cssSelector(".thread:nth-child(" + threadNumber + ") li.follow");
+    WebElement followButton = wait.forElementVisible(followButtonBy);
+
     if (followButton.getText().contains("Following")) {
       PageObjectLogging.log("unfollowIfDiscussionIsFollowed",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "unfollowIfDiscussionIsFollowed" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)